src/ServiceMonitor/ServiceMonitor.vcxproj

```diff
@@ -31,6 +26,10 @@
     <WindowsTargetPlatformVersion>10.0.15063.0</WindowsTargetPlatformVersion>
   </PropertyGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
+  <PropertyGroup>
+    <OutDir>$(RepositoryRoot)bin\$(Configuration)\$(PlatformShortname)\</OutDir>
+    <IntDir>$(RepositoryRoot)obj\$(PlatformShortname)\$(Configuration)\$(MSBuildThisFileName)\</IntDir>

```

Comment:
What's the usage of IntDir?

Reply:
IntDir is for intermediate directory. It is use for intermediate files produced during the build process like object files, temp files etc.

Reply:
I see. Thanks for the explanation.